### PR TITLE
Better plot demos for docs

### PR DESCRIPTION
**Pull request by aliabd**
I split up the plots from `native_plots` and added more code to them to try and make them as clear as possible. They're much longer now but I think that's fine as long as they're clear. 

Closes: #4179 

Happy to change them more if anyone has ideas

### CODE REVIEW

File: demo/line_plot/run.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#         ],
+#         "price": [random.randint(10, 200) for _ in range(120)],
+#     }
+# )
+
+
+def line_plot_fn(dataset):
+    if dataset == "stocks":
+        return gr.LinePlot.update(
+            fake_stocks,
```
Comment (freddyaboulton) [inline demo/line_plot/run.py:44]: fake_stocks not defined - I think you meant stocks?

**Comment by aliabd** [comment]
thanks @freddyaboulton, made the changes

File: demo/line_plot/run.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+with gr.Blocks() as line_plot:
+    with gr.Row():
+        with gr.Column():
+            dataset = gr.Dropdown(
+                choices=["stocks", "climate", "seattle_weather", "gapminder"],
+                value="stocks",
+            )
+        with gr.Column():
+            plot = gr.LinePlot(show_label=False)
```
Comment (freddyaboulton) [inline demo/line_plot/run.py:100]: Let's `show_label=True` for all of them? WDYT?

Comment (aliabd) [reply demo/line_plot/run.py:100]: sure yeah

**Comment by abidlabs** [comment]
Looks like everything is addressed, so I'll go ahead and merge this in

**Pull request merged**